### DISCUSSION TITLE
Are Macbooks with the M2 Pro and M2 Max chips supported?

### DISCUSSION THREAD

**Discussion by lyncasterc** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#896] [Q&A] [answered]
I see that the Macbook Pro 13-inch M2 is listed under the supported models but not the newer 14-inch and 16-inch models with the M2 Pro and M2 Max chips. Are these models currently supported or are there plans to eventually support them? Thanks!

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @lyncasterc, these models are currently not (yet) officially supported because we did not have a chance to test them yet. However, the plan is definitely to also have these new devices supported by AlDente.

Cheers,
Matthias
